routes/api/drivers.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+const router = new express.Router();
+const DriverService = require('../../services/DriverService');
+
+router.get('/drivers/:id', async (req, res) => {
```

Comment:
we can have validation schema, that will check, that id is uuid format,, that could be useful